addons/binding/org.openhab.binding.spotify/src/main/java/org/openhab/binding/spotify/internal/SpotifySession.java

```diff
@@ -0,0 +1,1299 @@
+                this.availableMarkets = availableMarkets;
+            }
+
+            public Integer getDiscNumber() {
+                return discNumber;
+            }
+
+            public void setDiscNumber(Integer discNumber) {
+                this.discNumber = discNumber;
+            }
+
+            public Long getDurationMs() {
+                return durationMs;
+            }
+
+            public void setDurationMs(Long durationMs) {
+                this.durationMs = durationMs;
+            }
+
+            public Boolean getExplicit() {
+                return explicit;
+            }
+
+            public void setExplicit(Boolean explicit) {
+                this.explicit = explicit;
+            }
+
+            public ExternalIds getExternalIds() {
+                return externalIds;
+            }
+
+            public void setExternalIds(ExternalIds externalIds) {
+                this.externalIds = externalIds;
+            }
+
+            public ExternalUrls getExternalUrls() {
+                return externalUrls;
+            }
+
+            public void setExternalUrls(ExternalUrls externalUrls) {
+                this.externalUrls = externalUrls;
+            }
+
+            public String getHref() {
+                return href;
+            }
+
+            public void setHref(String href) {
+                this.href = href;
+            }
+
+            public String getId() {
+                return id;
+            }
+
+            public void setId(String id) {
+                this.id = id;
+            }
+
+            public String getName() {
+                return name;
+            }
+
+            public void setName(String name) {
+                this.name = name;
+            }
+
+            public Integer getPopularity() {
+                return popularity;
+            }
+
+            public void setPopularity(Integer popularity) {
+                this.popularity = popularity;
+            }
+
+            public String getPreviewUrl() {
+                return previewUrl;
+            }
+
+            public void setPreviewUrl(String previewUrl) {
+                this.previewUrl = previewUrl;
+            }
+
+            public Integer getTrackNumber() {
+                return trackNumber;
+            }
+
+            public void setTrackNumber(Integer trackNumber) {
+                this.trackNumber = trackNumber;
+            }
+
+            public String getType() {
+                return type;
+            }
+
+            public void setType(String type) {
+                this.type = type;
+            }
+
+            public String getUri() {
+                return uri;
+            }
+
+            public void setUri(String uri) {
+                this.uri = uri;
+            }
+
+        }
+    }
+
+    /**
+     *
+     * @author Andreas Stenlund
+     *
+     */
+    public class SpotifyWebAPIDeviceList {
+
+        public class Device {
```

Comment:
Do these classes need to be inner classes of each other or can they just be static inner classes? And thus also just separate classes in there own files.